src/app/shared/stack-blitz/stack-blitz-writer.ts

```diff
@@ -203,7 +203,7 @@ export class StackBlitzWriter {
         .replace(/material-docs-example/g, data.selectorName)
         .replace(/\${title}/g, data.description);
     } else if (fileName === '.stackblitzrc') {
-      fileContent = fileContent.replace(/\${startCommand}/, isTest ? 'turbo test' : 'turbo start');
+      fileContent = fileContent.replace(/\${startCommand}/, isTest ? 'yarn test' : 'yarn start');
```

Comment:
I actually think we wanted `npm` right? The lock file is already a NPM lock file

Reply:
I converted the Stackblitz template to yarn since npm was timing out. See https://github.com/angular/material.angular.io/pull/1194

Reply:
Ah great! @d3lm what will be the Yarn support strategy with WebContainers? For angular.io we don't use Yarn

Reply:
@devversion How do you mean? For yarn we currently support yarn v1. Or maybe I am misunderstanding.